src/translations/index.ts

```diff
@@ -1,135 +1,73 @@
-  [LanguageKey.DeDe]: DeDe_TRANSLATIONS,
-  [LanguageKey.ItIt]: ItIt_TRANSLATIONS,
-  [LanguageKey.AfZz]: AfZz_TRANSLATIONS,
-  [LanguageKey.BgBg]: BgBg_TRANSLATIONS,
-  [LanguageKey.ZhCn]: ZhCn_TRANSLATIONS,
-  [LanguageKey.HrHr]: HrHr_TRANSLATIONS,
-  [LanguageKey.CsCz]: CsCz_TRANSLATIONS,
-  [LanguageKey.DaDk]: DaDk_TRANSLATIONS,
-  [LanguageKey.EnGb]: EnGb_TRANSLATIONS,
-  [LanguageKey.EnCa]: EnCa_TRANSLATIONS,
-  [LanguageKey.EnAe]: EnAe_TRANSLATIONS,
-  [LanguageKey.FiFi]: FiFi_TRANSLATIONS,
-  [LanguageKey.ElGr]: ElGr_TRANSLATIONS,
-  [LanguageKey.HiIn]: HiIn_TRANSLATIONS,
-  [LanguageKey.HuHu]: HuHu_TRANSLATIONS,
-  [LanguageKey.IdId]: IdId_TRANSLATIONS,
-  [LanguageKey.JaJp]: JaJp_TRANSLATIONS,
-  [LanguageKey.KoKr]: KoKr_TRANSLATIONS,
-  [LanguageKey.LtLt]: LtLt_TRANSLATIONS,
-  [LanguageKey.MsMy]: MsMy_TRANSLATIONS,
-  [LanguageKey.MrIn]: MrIn_TRANSLATIONS,
-  [LanguageKey.NbNi]: NbNi_TRANSLATIONS,
-  [LanguageKey.PlPl]: PlPl_TRANSLATIONS,
-  [LanguageKey.PtBr]: PtBr_TRANSLATIONS,
-  [LanguageKey.PtPt]: PtPt_TRANSLATIONS,
-  [LanguageKey.RoRo]: RoRo_TRANSLATIONS,
-  [LanguageKey.RuRu]: RuRu_TRANSLATIONS,
-  [LanguageKey.SrLatnRs]: SrLatnRs_TRANSLATIONS,
-  [LanguageKey.SvSe]: SvSe_TRANSLATIONS,
-  [LanguageKey.TaIn]: TaIn_TRANSLATIONS,
-  [LanguageKey.ThTh]: ThTh_TRANSLATIONS,
-  [LanguageKey.TrTr]: TrTr_TRANSLATIONS,
-  [LanguageKey.UkUa]: UkUa_TRANSLATIONS,
-  [LanguageKey.ViVn]: ViVn_TRANSLATIONS,
-  [LanguageKey.ZuZa]: ZuZa_TRANSLATIONS,
-  [LanguageKey.EnUS]: EnUS_TRANSLATIONS,
-  [LanguageKey.EnAu]: EnAu_TRANSLATIONS,
-  [LanguageKey.FrBe]: FrBe_TRANSLATIONS,
-  [LanguageKey.FrCa]: FrCa_TRANSLATIONS,
-  [LanguageKey.EnIe]: EnIe_TRANSLATIONS,
-  [LanguageKey.NlNl]: NlNl_TRANSLATIONS,
-  [LanguageKey.EsEs]: EsEs_TRANSLATIONS,
-  [LanguageKey.Es419]: Es419_TRANSLATIONS,
-  [LanguageKey.ZhHk]: ZhHk_TRANSLATIONS,
-};
+  LanguageKey.DeAt,
+  LanguageKey.DeCh,
+  LanguageKey.ItCh,
+  LanguageKey.FrCh,
+  LanguageKey.NlBe,
+  LanguageKey.HeIl,
+  LanguageKey.EnNz,
+  LanguageKey.ArAe,
+  LanguageKey.FrFr,
+  LanguageKey.DeDe,
+  LanguageKey.ItIt,
+  LanguageKey.AfZz,
+  LanguageKey.BgBg,
+  LanguageKey.ZhCn,
+  LanguageKey.HrHr,
+  LanguageKey.CsCz,
+  LanguageKey.DaDk,
+  LanguageKey.EnGb,
+  LanguageKey.EnCa,
+  LanguageKey.EnAe,
+  LanguageKey.FiFi,
+  LanguageKey.ElGr,
+  LanguageKey.HiIn,
+  LanguageKey.HuHu,
+  LanguageKey.IdId,
+  LanguageKey.JaJp,
+  LanguageKey.KoKr,
+  LanguageKey.LtLt,
+  LanguageKey.MsMy,
+  LanguageKey.MrIn,
+  LanguageKey.NbNi,
+  LanguageKey.PlPl,
+  LanguageKey.PtBr,
+  LanguageKey.PtPt,
+  LanguageKey.RoRo,
+  LanguageKey.RuRu,
+  LanguageKey.SrLatnRs,
+  LanguageKey.SvSe,
+  LanguageKey.TaIn,
+  LanguageKey.ThTh,
+  LanguageKey.TrTr,
+  LanguageKey.UkUa,
+  LanguageKey.ViVn,
+  LanguageKey.ZuZa,
+  LanguageKey.EnUS,
+  LanguageKey.EnAu,
+  LanguageKey.FrBe,
+  LanguageKey.FrCa,
+  LanguageKey.EnIe,
+  LanguageKey.NlNl,
+  LanguageKey.EsEs,
+  LanguageKey.Es419,
+  LanguageKey.ZhHk,
+];
```

Comment:
reminder to revert files in folder back to `translations/*.json` instead of `translations/languages/*.json`